public/locales/fr/translation.json

```diff
@@ -815,7 +815,7 @@
     "title": "Paramètres",
     "general": "Général",
     "actionItemCategories": "Catégories d'éléments d'action",
-    "updateOrganization": "Mettre à jour l'organisation",
+    "Edit Organization": "Éditer l'organisation",
```

Comment:
This is not in keeping with the camel case standard. Remove the space, use camel case and update the source code.